### PR TITLE
cmake: Add cython modules to run-tox test's PYTHONPATH

### PR DESCRIPTION
**Pull request by alimaredia**
Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

**Comment by alimaredia** [comment]
@tchaikov The error I'm getting from run-tox-ceph-disk is that the ceph command can't import rados. This didn't fix it but made me realize that the PYTHONPATH isn't the same as it is in add_ceph_test.

**Comment by tchaikov** [comment]
@alimaredia the run-tox-ceph-disk works for me once. will fix it tmr.

**Comment by alimaredia** [comment]
@tchaikov can we merge this?

**Comment by liewegas** [comment]
@alimaredia i think he's long since gone to bed.  doesn't seem right to have the arch and version string hard coded there though!

File: src/test/CMakeLists.txt [hunk 1]
```diff
@@ -537,7 +537,7 @@ set_property(TEST
   CEPH_DETECT_INIT_VIRTUALENV=${CEPH_DETECT_INIT_VIRTUALENV}
   LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib
   PATH=$ENV{PATH}:${CMAKE_RUNTIME_OUTPUT_DIRECTORY}:${CMAKE_SOURCE_DIR}/src
-  PYTHONPATH=${CMAKE_SOURCE_DIR}/src/pybind
+  PYTHONPATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cython_modules/lib.linux-x86_64-2.7:${CMAKE_SOURCE_DIR}/src/pybind
```
Comment (liewegas) [inline src/test/CMakeLists.txt]: lib.$(uname -s)-$(uname -m)-2.7
?

Comment (dang) [inline src/test/CMakeLists.txt]: ...-$(PYTHON_VERSION_MAJOR).$(PYTHON_VERSION_MINOR) ?

**Comment by tchaikov** [comment]
superseded by #9919.

**Pull request closed without merging**